Automattic/themes
# Seedlet: Editor titles

The display of Post titles in Seedlet are the same in the frontend and the editor, but in Spearhead they aren't. This PR tidies the way that post titles are styled in Seedlet which also has the effect that in Spearhead the editor now matches the frontend:

To test
- switch your blog to Spearhead
- edit a post in the editor
- Check that the post title looks like this:

<img width="1107" alt="Screenshot 2020-10-06 at 14 22 22" src="https://user-images.githubusercontent.com/275961/95207071-5af46680-07df-11eb-9e0a-34781012ba70.png">


Looking at this makes me wonder if we should just use the main styles in the editor too, rather than having a different stylesheet.
